Add new features

JulianPrieber opened the pull request:
**Add features:**
- custom button for custom links
- rounded corners avatar image littlelink.blade.php
- Social Share Preview for individual users [see]
- button hover animations [demo]

See commits: [main branch]

   [demo]: <https://julianprieber.github.io/littlelink-admin-demo/>
   [see]: <https://github.com/JulianPrieber/littlelink-admin/commit/0750e48830ccad6e431e147bf81e4026a103dec5#comments>
   [main branch]: <https://github.com/JulianPrieber/littlelink-admin/commits/main>

JulianPrieber committed:
Added rounded avatar

Added class 'rounded-avatar' to avatar images

JulianPrieber committed:
Added class 'rounded-avatar'

Added class 'rounded-avatar' with border-radius set to 50%.

Remove or uncomment this if you don't want rounded avatars for users.

JulianPrieber committed:
Replaced (-) hyphen button seed with custom link button

Replaced (-) hyphen button seed with custom link button as first button

JulianPrieber committed:
Added custom link button

Added custom link button to allow users to add their own custom links

JulianPrieber committed:
Added custom link icon

Added white icon for custom links

JulianPrieber committed:
Update README.md

JulianPrieber committed:
Update README.md

JulianPrieber committed:
Update README.md

JulianPrieber committed:
Added support for social media preview

This shows a preview for Name, Bio and avatar image of users profiles if shared on social sites like Facebook,  Twitter, Facebook Messenger, WhatsApp, Discord, Pinterest and many more.

JulianPrieber committed:
Update README.md

JulianPrieber committed:
Update README.md

JulianPrieber committed:
merged

merged

JulianPrieber committed:
Added Hover.css

Added hover-min.css by Ian Lunn (https://github.com/IanLunn/Hover) to allow use of animations with buttons and icons.

JulianPrieber committed:
Edit LICENSE

Added license for the use of Hover.css by Ian Lunn (https://store.ianlunn.co.uk/licenses/personal).

See: https://github.com/JulianPrieber/littlelink-admin/commit/0bb11a1e66f465a2c0e65a97217bfe65f624a55f

JulianPrieber committed:
Added button animation littlelink

Added button animation to littlelink.blade.php.

  -  Included hover-min.css for animation
  -  Added class 'button hvr-grow' to all buttons
  -  Added class 'hvr-icon-wobble-vertical' to all buttons (for icon animation)
  -  Added class 'hvr-icon' to all icons (to indicate icon to hover-min.css)

Read more on how to use and change animations here: https://github.com/IanLunn/Hover/blob/master/README.md
Try a demo of the added animations here: https://julianprieber.github.io/littlelink-admin-demo/

JulianPrieber committed:
Revert "Added button animation littlelink"

This reverts commit 84bfe583a6cc44809504830d2469e94ed01a7c53.

JulianPrieber committed:
Added button animation littlelink

Added hover button animation to littlelink.blade.php.

  -  Included hover-min.css for animation
  -  Added class 'button hvr-grow' to all buttons
  -  Added class 'hvr-icon-wobble-vertical' to all buttons (for icon animation)
  -  Added class 'hvr-icon' to all icons (to indicate icon to hover-min.css)

Read more on how to use and change animations here: https://github.com/IanLunn/Hover/blob/master/README.md
Try a demo of the added animations here: https://julianprieber.github.io/littlelink-admin-demo/

JulianPrieber committed:
Added button animation home

Added hover button animation to home.blade.php.

  -  Included hover-min.css for animation
  -  Added class 'button hvr-grow' to all buttons
  -  Added class 'hvr-icon-wobble-vertical' to all buttons (for icon animation)
  -  Added class 'hvr-icon' to all icons (to indicate icon to hover-min.css)

Read more on how to use and change animations here: https://github.com/IanLunn/Hover/blob/master/README.md
Try a demo of the added animations here: https://julianprieber.github.io/littlelink-admin-demo/

JulianPrieber committed:
Update README.md

Added partner button animations (https://github.com/IanLunn/Hover)

JulianPrieber committed:
Update README.md

Added feature button hover animation + demo (https://julianprieber.github.io/littlelink-admin-demo)

JulianPrieber committed:
Merge pull request #2 from JulianPrieber/main

merged

latuminggi commented:
wow, i will look into it first before merge your PR
thx a lot for your contributions

JulianPrieber committed:
fixed hover-min.css

Moved hover-min.css from public/css to public/littlelink/css

JulianPrieber committed:
fixed hover-min.css

Moved hover-min.css from public/css to public/littlelink/css

JulianPrieber committed:
fixed URL

JulianPrieber committed:
fixed spelling

changed button to buttons

JulianPrieber committed:
Made link title optional

Made link title (previously required) optional when adding personal LittleLink links on the User/Admin Panel.
If the 'Title' field is left empty, the button name is now used instead of rejecting the addition.

JulianPrieber committed:
Revert "Made link title optional"

This reverts commit 2b47f5858d78be272db495533e667a69ebdbe284.
